packages/icon-build-helpers/src/metadata/extensions/icons.js

```diff
@@ -47,9 +47,7 @@ const icons = {
       const icon = registry.get(entry.name);
 
       // Add namespace information for the icon
-      if (Array.isArray(icon.namespace) && icon.namespace.length > 0) {
-        entry.namespace = icon.namespace.join('/');
-      }
+      entry.namespace = Array.isArray(icon.namespace) ? icon.namespace : [];
```

Comment:
Just a quick heads up for metadata changes @vpicone just going to have this be an array instead of converting it 👍